css/style.css

```diff
@@ -0,0 +1,64 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+html, body {
```

Comment:
You can add the css property `margin: auto` to give some spacing between everything